guidelines/goals.md

```diff
@@ -0,0 +1,69 @@
+# Goals and Non-Goals
+
+This document defines the purpose of the Message Format Working Group (MFWG)
+and informs the decisions about the scope and the priorities of its efforts.
+
+## Goals
+
+The primary task of the MFWG is to develop an industry standard for the
+representation of localizable message strings. The design goals are listed
+below.
+
+ 1. Express grammatical features, such as plurals, genders, and inflections.
+
+ 2. Express other variance in translation, due to linguistic and regional
+    features, the presentation media, context, circumstance, and other factors.
+
+ 3. Represent structured data alongside translations, such as markup, comments,
+    and metadata.
+
+ 4. Ensure interoperability with existing interchange formats, in particular
+    with XLIFF.
+
+ 5. Ensure that the standard can integrate with existing TMS and CAT
```

Comment:
Should we explain what the abbreviations stand for?

```suggestion
 5. Ensure that the standard can integrate with existing TMS (Translation
    Management System) and CAT (Computer Assisted Translation)
```

Reply:
Just noticed these are included in the [glossary](https://github.com/unicode-org/message-format-wg/pull/80/files), so perhaps not needed

Reply:
probably we can just link to this expressions to the  glossary